sandpack-react/src/components/Tests/Specs.tsx

```diff
@@ -138,36 +143,44 @@ export const Specs: React.FC<Props> = ({
               )}
 
               <FilePath
-                onClick={(): void => openSpec(spec.name)}
+                onClick={(): void => {
+                  if(!hideFailingTests){
+                    openSpec(spec.name)
+                  }
+                }}
                 path={spec.name}
               />
             </div>
 
-            {verbose && <Tests tests={tests} />}
+            {verbose &&!hideFailingTests && <Tests tests={tests} />}
```

Comment:
Could you please run the prettier script? A lot of files are not correctly formatted